sharing/src/test/scala/io/delta/sharing/spark/DeltaFormatSharingSourceSuite.scala

```diff
@@ -49,6 +50,11 @@ class DeltaFormatSharingSourceSuite
 
   import testImplicits._
 
+  override def beforeAll(): Unit = {
```

Comment:
this seems like something you can have in the common test trait: `DeltaSharingDataSourceDeltaTestUtils `

Reply:
good point, moved.